### PR TITLE
Issue #9 | Hover effect on global Buttons

### PR DESCRIPTION
**Pull request by Niaz-Ul-Haque**
[#9](https://github.com/jrobind/dev-jot/issues/9)
I did as mentioned in the issue. Please do let me know if there's anything that I missed out. Thank you for the opportunity to contribute. Enjoyed it very much!.

### CODE REVIEW

**Review by jrobind — COMMENTED** [review]
Hey @Niaz-Ul-Haque thanks for sending over a PR :) 

Could you make sure indentation is 2 spaces across all files, please? Because it's 4 spaces the code diff is really large for this PR so it's hard to see what actual changes you've made.

Let me know if that makes sense.

**Comment by Niaz-Ul-Haque** [comment]
Ahhhh, Ill do that. Thanks for pointing it out. I'll fix it asap.

**Review by Niaz-Ul-Haque — COMMENTED** [review]
Review done, requested an updated PR. Thank you!

File: public/css/style.css [hunk 1]
```diff
@@ -25,7 +27,10 @@ body {
   border: none;
   cursor: pointer;
   font: inherit;
-  box-shadow: 0 3px 6px 0 rgba(0,0,0,.05), 0 3px 6px 0 rgba(0,0,0,.05);
+  box-shadow: 0 3px 6px 0 rgba(0, 0, 0, 0.05), 0 3px 6px 0 rgba(0, 0, 0, 0.05);
+}
+.button:hover {
```
Comment (jrobind) [inline public/css/style.css:32]: We already have `cursor:pointer` applied above and we don't really want to override this. I was thinking of a hover effect that would alter the button color, opacity, and/or apply a small animation.

Comment (Niaz-Ul-Haque) [reply public/css/style.css:32]: Ahh, now I get what you meant. I'll be sure to fix them in a few days as I currently have finals for 3 consecutive days. Don't worry and thank you for pointing it out!

Comment (jrobind) [reply public/css/style.css:32]: You're welcome! No rush at all, take your time with this. Good luck with your exams 😄

**Comment by jrobind** [comment]
I'm going to close this PR and open the issue back up for someone else to pick up.

**Pull request closed without merging**